offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferView.groovy

```diff
@@ -206,6 +209,14 @@ class CreateOfferView extends FormLayout{
                 this.overviewView)
     }
 
+    void resetViewContent() {
+        viewHistory.loadNewView(this.projectInformationView)
+        navigationView.rewind()
+        projectInformationView.reset()
+        customerSelectionView.reset()
+        projectManagerSelectionView.reset()
+    }
+
```

Comment:
what about resetting the ItemsView and the OverviewView?

Reply:
the overview does not need to be reset, as it is a consequence of the previous input and based on the shared model.

The ItemsView I can still add of course.